### PR TITLE
Add cassandra connector entry

### PR DESCRIPTION
**Pull request by Setogit**
#### Related issues

strongloop-internal/scrum-asteroid#23

- None

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: available-connectors.json [hunk 1]
```diff
@@ -225,6 +225,49 @@
...
+        "type": "string",
+        "display": "password"
+      },
+      "database": {
+        "type": "string"
+      },
+      "connectTimeout": {
+        "type": "number",
+        "default": 30000,
+        "description": "connectTimeout(msec)"
```
Comment (superkhau) [inline available-connectors.json]: The connection timeout value in milliseconds -- `connectTimeout(ms)`.

File: available-connectors.json [hunk 2]
```diff
@@ -225,6 +225,49 @@
...
+      },
+      "readTimeout": {
+        "type": "number",
+        "default": 30000,
+        "description": "readTimeout(msec)"
```
Comment (superkhau) [inline available-connectors.json]: The read timeout value in milliseconds -- `readTimeout(ms)`.

File: available-connectors.json (hunk 1 above)
Comment (Setogit) [inline available-connectors.json]: done

File: available-connectors.json (hunk 2 above)
Comment (Setogit) [inline available-connectors.json]: done

**Pull request merged**